dev/domain-controller/README.md

```diff
@@ -1,50 +1,88 @@
+  * This account is only required to deploy the system. During deployment it will create an application in the Azure Active Directory account associated with the current Azure subscription. The application name is 'CAM-\<resourceGroupName\>'. It will also create a Service Principal account as part of this application which has contributor access to the resource group it is being deployed to. After deployment, only the Service Principal account is used for interaction with Azure API's.
 * AzureAdminPassword: The password of the Azure account with **owner** access to the subscription.
 * registrationCode: The license registration code for the PCoIP CAS licenses.
 * adminVMBlobSource: The location of the blobs for admin GUI machine installation. Use the default unless you are specifically deploying with modified binaries.
 * \_artifactsLocation: The location of resources, such as templates and DSC modules, that the template depends on. Use the default unless you are specifically deploying with modified templates or binaries.
 * \_artifactsLocationSasToken: - an auto-generated token to access _artifactsLocation. If _artifactsLocation does not need an access token (which is the default) then this can be blank.
 
-## Known issues with deploying the solution
+ ## Deployment Template 
+ This section will enable you to perform a deployment using the deployment parameters set out above.
+ This template will deploy 5 virtual machines (along with a new VNet, Storage Account, Load Balancer, Azure KeyVault, and Gateway).
+
+ To administer the deployment through the Cloud Access Manager GUI, https: to the public IP of the applicationGateway1 Application Gateway. To connect to the pre-created Agent virtual machine, point the PCoIP client to the public IP of the applicationGateway1 Application gateway and login with the administrator credentials. To manage the Active Directory Domain, RDP to the public IP address of vm-dc (the domain controller).
+
+The following steps outline the procedure for performing a deployment of CAM using Microsoft Azure: 
+
+ Click the <b> Deploy Azure </b> button to  begin.
+
+<a target="_blank" href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fteradici%2Fdeploy%2Fmaster%2Fdev%2Fdomain-controller%2Fazuredeploy.json">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+1. Select the Microsoft Azure account you want to access.
+1. Enter your Password and click <b>Sign in</b>
+1. ON the Customized Template page create a new Resource group by selecting the <b>Create New</b> icon and entering a name for the group.
+* You can also select a pre-defined Resource group by selecting the <b>Use Existing</b> icon and clicking on one of the groups from the dropdown menu.
+
+4. Select a location from the dropdown menu.
```

Comment:
Should we link to the note below on geographic limitations for the Graphics Agent machines?